### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Fixes misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/cpp-peglib/actions/runs/7383303958/attempts/1#summary-20084279619

The action will report that the changes in this PR would make it happy: https://github.com/jsoref/cpp-peglib/actions/runs/7383304003/attempts/1#summary-20084280009

### CODE REVIEW

File: .github/workflows/cmake.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ jobs:
       working-directory: ${{runner.workspace}}/build
       # Note the current convention is to use the -S and -B options here to specify source
       # and build directories, but this is only available with CMake 3.13 and higher.
-      # The CMake binaries on the Github Actions machines are (as of this writing) 3.12
+      # The CMake binaries on the GitHub Actions machines are (as of this writing) 3.12
```
Comment (jsoref) [inline .github/workflows/cmake.yml:35]: brand

File: README.md [hunk 2]
```diff
@@ -59,8 +59,8 @@ int main(void) {
   // (2) Make a parser
   parser parser(R"(
     # Grammar for Calculator...
-    Additive    <- Multitive '+' Additive / Multitive
-    Multitive   <- Primary '*' Multitive / Primary
+    Additive    <- Multiplicative '+' Additive / Multiplicative
+    Multiplicative   <- Primary '*' Multiplicative / Primary
```
Comment (jsoref) [inline README.md:63]: I could adjust the whitespace...

File: example/calc.cc [hunk 3]
```diff
@@ -9,29 +9,29 @@ int main(void) {
...
         # Grammar for Calculator...
-        Additive    <- Multitive '+' Additive / Multitive
-        Multitive   <- Primary '*' Multitive^cond / Primary
+        Additive    <- Multiplicative '+' Additive / Multiplicative
+        Multiplicative   <- Primary '*' Multiplicative^cond / Primary
         Primary     <- '(' Additive ')' / Number
         Number      <- < [0-9]+ >
         %whitespace <- [ \t]*
-        cond <- '' { error_message "missing multitative" }
+        cond <- '' { error_message "missing multiplicative" }
```
Comment (jsoref) [inline example/calc.cc:17]: this item didn't match the items above...

**Comment by yhirose** [comment]
@jsoref I am so embarrassed (English isn't my first language. 😅). Thanks for all the corrections!

**Comment by jsoref** [comment]
Don't be. Projects by native speakers are as bad or worse.

**Pull request merged**